translations/strings.yaml

```diff
@@ -82,3 +98,172 @@ splittunnel:
     comment:
       - Displayed in notification after the user added a missing app to the split-tunnel List; %1 is the name of the added application
 
+  excellentLabel: Excellent
+  #Screen 843
+  selectionHeader: Give feedback
+  reviewHeader: Give us 5 stars!
+  reviewBody: If you enjoy using Mozilla VPN, would you review it? Thanks for your support!
+  leaveReviewButton: Leave a Review
+  skipLink: Skip
+
+#Additional messaging for network issues with split tunneling settings
+splitTunnelNetworkMessaging:
+  infoText: For any questions about setting app permissions, we've provided some helpful tips on our website.
```

Comment:
Please double check `'` and `"`, using `’` and `“”` instead. The linter right now is failing before reaching that point
https://github.com/mozilla-mobile/mozilla-vpn-client/pull/1573/checks?check_run_id=3355539684